Teamleader.php

```diff
@@ -414,6 +414,28 @@ public function crmUpdateContact(
     }
 
     /**
+     * Delete a contact
+     *
+     * @todo    N/A
```

Comment:
I added the "todo" in analogy with other methods.

I could remove it if you like. There really is no other reason it's there.